Xamarin.Forms.Platform.iOS/ModalWrapper.cs

```diff
@@ -14,10 +14,10 @@ internal ModalWrapper(IVisualElementRenderer modal)
 			_modal = modal;
 
 			var elementConfiguration = modal.Element as IElementConfiguration<Page>;
-			if (elementConfiguration?.On<PlatformConfiguration.iOS>().ModalPresentationStyle() == PlatformConfiguration.iOSSpecific.UIModalPresentationStyle.FormSheet)
-				ModalPresentationStyle = UIKit.UIModalPresentationStyle.FormSheet;
+			if (elementConfiguration?.On<PlatformConfiguration.iOS>()?.ModalPresentationStyle() is PlatformConfiguration.iOSSpecific.UIModalPresentationStyle style)
+				ModalPresentationStyle = style.ToNativeModalPresentationStyle();
 
-			View.BackgroundColor = UIColor.White;
+			View.BackgroundColor = UIColor.Clear;
```

Comment:
I think this needs to stay as White if the Style requested is FullScreen otherwise it changes the look of a Full Screen Modal that has a transparent background.